src/require.ts

```diff
@@ -15,11 +15,14 @@ export function jitiRequire(
 ) {
   const cache = ctx.parentCache || {};
 
-  // Check for node: and file: protocol
+  // Check for node:, file:, and data: protocols
   if (id.startsWith("node:")) {
     id = id.slice(5);
   } else if (id.startsWith("file:")) {
     id = fileURLToPath(id);
+  } else if (id.startsWith("data:")) {
+    debug(ctx, "[native]", "[data]", opts.async ? "[import]" : "[require]", id);
+    return nativeImportOrRequire(ctx, id, opts.async);
```

Comment:
I think this is only supported via async import (CJS does not support `data:`)

Reply:
oh! You're right. What should the behaviour be if `opts.async === true` then? Should we ignore it and always use the async `import`, error out, or something else?

Reply:
Yes probably we can throw an error when async is false to say use import/jiti.import

Reply:
Done